resources/views/abilities/entities/_form.blade.php

```diff
@@ -2,13 +2,9 @@
 {{ csrf_field() }}
 
 <x-grid type="1/1">
-    @include('cruds.fields.entity', [
-        'required' => true,
-        'route' => 'search.ability-entities',
-        'placeholder' => __('entities/relations.placeholders.target'),
-        'preset' => false,
-        'dropdownParent' => request()->ajax() ? '#primary-dialog' : null,
-    ])
+    <x-forms.field field="entities" :required="true" >
+        @include('components.form.entities', ['options' => ['exclude-entity' => $model->entity->id]])
```

Comment:
components shouldn't be included in blade views but use laravel's components feature <x-entities>